pkgs/development/python-modules/rfc3986/default.nix

```diff
@@ -10,9 +10,9 @@ buildPythonPackage rec {
     sha256 = "1qf4dyxvjs7mxrxc0gr7gzyn4iflb2wgq01r5pzrxac8rnvy8fmw";
   };
 
-  buildInputs = [ pytest ];
+  checkInputs = [ pytest ];
   checkPhase = ''
-    py.test
```

Comment:
unnecessary change

Reply:
ah, nevermind, I did not know `pytest` was the preferred invocation.

Reply:
Indeed, but might as well change it right?